web_drop_target/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
not blocking: png would be better

Reply:
the [mtq template](https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md)  disagrees